drivers/bridge/setup_device.go

```diff
@@ -31,7 +32,29 @@ func setupDevice(config *NetworkConfiguration, i *bridgeInterface) error {
 	}
 
 	// Call out to netlink to create the device.
-	return netlink.LinkAdd(i.Link)
+	if err = netlink.LinkAdd(i.Link); err != nil {
+		return types.InternalErrorf("Failed to program bridge link: %s", err.Error())
+	}
+
+	if config.FixedCIDRv6 != nil {
+		// Setting route to global IPv6 subnet
+		log.Infof("Adding route to IPv6 network %s via device %s", config.FixedCIDRv6.String(), config.BridgeName)
```

Comment:
I have not found other log to Info level in bridge.
Any reason why this action should log to info ?